app/routes.py

```diff
@@ -24,10 +24,10 @@ async def root(request):
 def setup(app, url_path_prefix):
   """Set up routes as resources so we can use the `Index:get` notation for URL lookup."""
 
-  path = str(Path(__file__).resolve()) + "/app/"
-  path = "/opt/ui/app/assets/"
+  p = Path("/tmp/assets/")
+  p.mkdir(parents=True, exist_ok=False)
```

Comment:
This should probably be `exist_ok=True`, just in case we restart the server locally. We want it to pass fine if it exists.
Linux convention is for the path in tmp to include the user, as well as the program, just in case some other program uses /tmp/assets. We know there's only one important functioning user on the system, so /tmp/fsdrui_assets would be perfectly fine.

Reply:
Have adjusted for exist_ok=True

Reply:
Have also adjusted   the tmp directory